src/jvmMain/kotlin/io/beatmaps/api/users.kt

```diff
@@ -450,22 +446,7 @@ fun Route.userRoute() {
     }
 
     get<UsersApi.Alerts> {
-        requireAuthorization { user ->
-            val targetId = if (it.id != null && user.isAdmin()) {
-                it.id
-            } else {
-                user.userId
-            }
-
-            val alerts = transaction {
-                ModLog.join(Beatmap, JoinType.INNER, Beatmap.id, ModLog.opOn).select {
-                    (Beatmap.uploader eq targetId) and
-                        (ModLog.type inList listOf(ModLogOpType.Unpublish, ModLogOpType.Delete).map { it.ordinal })
-                }.orderBy(ModLog.opAt, SortOrder.DESC).limit(30).map { Alert.from(it, cdnPrefix()) }
-            }
-
-            call.respond(alerts)
-        }
+        call.respondRedirect("/api/alerts/unread" + if (it.id != null) "/${it.id}" else "")
```

Comment:
What purpose does this redirect serve? The new response is mostly incompatible so any users calling this endpoint would receive an error anyway

Reply:
I figured we might not want to remove routes, but I'll get rid of it if it's not necessary.

Reply:
If we wanted to keep it we should keep the response here compatible but as it's an internal route I'd delete it.